src/experiment/metrics/basicmetrics.jl

```diff
@@ -14,10 +14,11 @@ It satisfies the two AbstractMetrics requirements:
 
     heuristic::H                                                        ->  The related heuristic for which the metrics stores the results.
     nodeVisited::Vector{Vector{Int64}}                                  ->  contains the result of each search in term of node visited : number of nodes visited to find every final state found (Solution / Infeasible case) reached during a search.
+    timeNeeded::Vector{Vector{Float32}}                                  ->  contains the result of each search in term of time needed: time needed to find every final state found (Solution / Infeasible case) reached during a search.
```

Comment:
added the time needed to find **every** final state found (Solution / Infeasible case) reached during a search, this is different from the `TotalTimeNeeded` that stores the computing time required to complete each search.